src/diffs/package-json.js

```diff
@@ -0,0 +1,43 @@
+// Heavily inspired by https://github.com/greenkeeperio/greenkeeper/blob/8be4bfc45dc991ec0c3331ac15b688728f5938a5/lib/diff-package-json.js
+
+const _ = require('windbreaker-service-util/util/objectHelpers')
```

Comment:
Since this is mostly a passthrough, I continued to use _ here. I'm okay with changing it if people feel strongly.